### ISSUE TITLE
removes `!important` declarations

### ISSUE THREAD

**Issue by pyramation** [postcss/postcss-js#19]
Seems that the default behavior removes any `!important` declarations (which isn't what I need):

```js
var postcss = require('postcss');
var postcssJs = require('postcss-js');

const text = `
.foo {
    margin-bottom: 0 !important
}
`;

let root = postcss.parse(text);
var cssObj = postcssJs.objectify(root);
postcss()
  .process(cssObj, { parser: postcssJs })
  .then(r=>{
    console.log(r.css);
  });
```

the output:

```css
.foo {
    margin-bottom: 0
}
```

let me know if I'm misunderstanding something or if there is a configuration, I did take a look at the src and couldn't find anything, thanks in advance!

**Comment by ai** [comment]
Yeap. It is a bug. Do you want to send PR or it is OK to wait few until I will fix it?

**Comment by pyramation** [comment]
I'll see what I can do :)

**Comment by ai** [comment]
You need to set `Declaration#important` flag on `!important` in parser

**Comment by ai** [comment]
I could do it today, but it is good task for PR, easy and fast :)

**Comment by pyramation** [comment]
ok, I have something that works with tests, but I edited objectify https://github.com/pyramation/postcss-js/commit/bb11c835b5b648a32127e9f742378611b283f6cb

If this looks OK, I'll make a PR. In meantime I'll checkout the parser

**Comment by ai** [comment]
Please pull new changes from `master`. I fixed old tests (PostCSS 5.1 or 5.2 broke it).

**Comment by pyramation** [comment]
👍  https://github.com/postcss/postcss-js/pull/20

**Comment by ai** [comment]
Thanks! Wait a second, I will release it after 30 minutes.

**Comment by ai** [comment]
Do you have Twitter account? (for tweet about new release)

**Comment by ai** [comment]
Released in 0.2
